File: sams-core-0.0.8/src/com/narendra/sams/core/service/InstallmentService.java
package com.narendra.sams.core.service;

import com.narendra.sams.core.domain.Installment;
import java.util.List;

public interface InstallmentService {
    List<Installment> getAllActiveInstallments();

    List<Installment> getInstallments(Long l);
}

File: sams/WebContent/sams/components/core/installment.service.js

samsApp.service('installmentService', function(ajaxService) {

	var resourceURL = _appContextPath+"/ws/installment";
	
	this.getInstallments = function(successCallBackFunction, errorCallBackFunction){
		return ajaxService.get(resourceURL+"/list", successCallBackFunction, errorCallBackFunction);
	}
	
})










